iphone/cli/commands/_build.js

```diff
@@ -4892,1243 +4891,1192 @@ iOSBuilder.prototype.writeDebugProfilePlists = function writeDebugProfilePlists(
+						const jsFiles = {};
+						const inputFiles = [];
+						for (let [ key, value ] of gatheredResults.jsFiles) {
+							jsFiles[key] = value;
+							inputFiles.push(value.src);
 						}
-
-						this.currentBuildManifest.files[file] = {
-							hash:  contents === null && prev ? prev.hash  : hash || this.hash(contents || ''),
-							mtime: contents === null && prev ? prev.mtime : srcMtime,
-							size:  contents === null && prev ? prev.size  : srcStat.size
-						};
-
-						this.unmarkBuildDirFile(info.dest);
-					}, this);
-				},
-			], next);
-		},
-
-		function copyCSSFiles() {
-			this.logger.debug(__('Copying CSS files'));
-			Object.keys(cssFiles).forEach(function (file) {
-				const info = cssFiles[file];
-				if (this.minifyCSS) {
-					this.logger.debug(__('Copying and minifying %s => %s', info.src.cyan, info.dest.cyan));
-					const dir = path.dirname(info.dest);
-					fs.ensureDirSync(dir);
-					fs.writeFileSync(info.dest, new CleanCSS({ processImport: false }).minify(fs.readFileSync(info.src).toString()).styles);
-				} else if (!this.copyFileSync(info.src, info.dest, { forceCopy: unsymlinkableFileRegExp.test(path.basename(file)) })) {
-					this.logger.trace(__('No change, skipping %s', info.dest.cyan));
-				}
-				this.unmarkBuildDirFile(info.dest);
-			}, this);
-		},
-
-		function (next) {
-			series(this, [
-				function processJSFiles(next) {
-					this.logger.info(__('Processing JavaScript files'));
-					const sdkCommonFolder = path.join(this.titaniumSdkPath, 'common', 'Resources', 'ios');
-					const task = new ProcessJsTask({
-						inputFiles: Object.keys(jsFiles).map(relPath => jsFiles[relPath].src),
-						incrementalDirectory: path.join(this.buildDir, 'incremental', 'process-js'),
-						logger: this.logger,
-						builder: this,
-						jsFiles,
-						jsBootstrapFiles,
-						sdkCommonFolder,
-						defaultAnalyzeOptions: {
-							minify: this.minifyJS,
-							transpile: this.transpile,
-							sourceMap: this.sourceMaps,
-							resourcesDir: this.xcodeAppDir,
+						const task = new ProcessJsTask({
+							inputFiles,
+							incrementalDirectory: path.join(this.buildDir, 'incremental', 'process-js'),
 							logger: this.logger,
-							targets: {
-								ios: this.minIosVersion
+							builder: this,
+							jsFiles,
+							jsBootstrapFiles,
+							sdkCommonFolder,
+							defaultAnalyzeOptions: {
+								minify: this.minifyJS,
+								transpile: this.transpile,
+								sourceMap: this.sourceMaps,
+								resourcesDir: this.xcodeAppDir,
+								logger: this.logger,
+								targets: {
+									ios: this.minIosVersion
+								}
 							}
-						}
-					});
-					task.run()
-						.then(() => {
-							this.tiSymbols = task.data.tiSymbols;
-
-							return next(); // eslint-disable-line promise/no-callback-in-promise
-						})
-						.catch(e => {
-							this.logger.error(e);
-							process.exit(1);
 						});
-				},
+						task.run()
```

Comment:
<!--
  0 failure: 
  2 warning:  iphone/cli/comman..., iphone/cli/comman...
  
  
  DangerID: danger-id-Danger;
  File: iphone/cli/commands/_build.js;
  Line: 5984;
-->

- :warning: iphone/cli/commands/_build.js line 5984 – Avoid using promises inside of callbacks. (promise/no-promise-in-callback)
- :warning: iphone/cli/commands/_build.js line 5984 – Avoid using promises inside of callbacks. (promise/no-promise-in-callback)